shader/draw_leaf.wgsl

```diff
@@ -73,7 +73,7 @@ fn main(
     workgroupBarrier();
     var m = sh_scratch[0];
     workgroupBarrier();
-    let tag_word = scene[config.drawtag_base + ix];
+    let tag_word = scene[min(config.drawtag_base + ix, config.drawtag_base + config.n_drawobj - 1u)];
```

Comment:
I think a better approach (and a pattern followed elsewhere) is to conditionalize the read so it becomes `DRAWTAG_NOP`.